cmd/api/src/analysis/ad/adcs_integration_test.go

```diff
@@ -109,6 +96,9 @@ func TestADCSESC1(t *testing.T) {
 				require.True(t, results.Contains(harness.ADCSESC1Harness.Group42))
 				require.True(t, results.Contains(harness.ADCSESC1Harness.Group43))
 
+				//Group47 has Enroll on EnterpriseCA1 that has a valid chain to Domain1, a domain that is marked as not collected
+				require.True(t, results.Contains(harness.ADCSESC1Harness.Group47))
```

Comment:
This is the relevant assertion for the fix in this changeset. The other changes are made in order to align testing set up with what actually happens during post processing.